spec/controllers/allocations_controller_spec.rb

```diff
@@ -167,7 +167,26 @@
     end
 
     context 'without DeliusDataJob' do
-      render_views
```

Comment:
think this was removed by accident?

Reply:
don't think this line should have been deleted

Reply:
I've put it back

Reply:
I can see that this allocation fixture `before` block has been moved one level up into the parent `context 'without DeliusDataJob'` block, where it used to be inside the child `context 'with an allocation'` block.

It looks like we create a different allocation fixture in the other child context block `context 'with a different allocation'`, so I don't think it uses the first allocation fixture.

I'm wondering if it makes more sense to keep it in the `context 'with an allocation'` block. Or is there are reason for it being moved up into the parent?